test_main.py

```diff
@@ -0,0 +1,154 @@
+import os
+import sys
+from fastapi.testclient import TestClient
+import pytest
+import random
+import string
+from app.main import app
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+client = TestClient(app)
+
+""" Testing all Gets"""
+
+def test_get_bot_scripts():
+    response = client.get("/select-all-from-bot-scripts")
+    assert response.status_code == 200
+
+def test_frankenbert():
+    test = 'A women walked her dog down the street and cop waved to her'
+    response = client.get(f'/frankenbert/{test}')
+    assert response.status_code == 200
+    response_body = response.json()
+    assert 'Rank' in response_body['Rank'] 
+    assert response_body['Rank'][5] in ['0', '1', '2', '3', '4', '5']
+    assert '%' in response_body['Confidence'] 
+
+
+def test_view_data():
+    response = client.get('/view-data')
+    assert response.status_code == 200
+    response_body = response.json()
+    assert len(response_body) == 5000
+
+
+def test_to_approve():
+    response = client.get('/view-data')
+    assert response.status_code == 200
+
+
+def test_advance_all():
+    response = client.get('/advance-all')
+    assert response.status_code == 200
+
+
+"""Test all Posts"""
+
+#yes
+def test_form_in():
+    response = client.post(
+        '/form-in/',
+        json={
+                "city": "cHicago",
+                "state": "ILLInoIs",
+                "confidence": random.random(),
+                "description": random_gen(0, 1000),
+                "force_rank": "1",
+                "incident_date": "09/09/2021", 
+                "incident_id": 1224, #Most respect foriegn key constraint
+                "lat": random.randint(0, 50),
+                "long": random.randint(0, 50),
+                "src": [f"https://twitter.com/{random_gen(0,20)}"],
+                "status": "approved",
+                "title": random_gen(0, 20),
+                "tweet_id": random_gen(1, 19),
+                "user_name": random_gen(0, 25)
+        } 
```

Comment:
The test may be useful for the stakeholder's tech team. I could keep it in a separate branch with all the necessary documentation, in case the client wants it.